crates/vision/src/line_detection.rs

```diff
@@ -112,10 +113,10 @@ impl LineDetection {
                 continue;
             }
             let (start_point_in_image, start_point_in_robot) =
-                match points_with_projection_onto_line.iter().find_map(
+                match points_with_projection_onto_line.iter().copied().find_map(
```

Comment:
You are now copying all the points vs. only the found point. Does that have an impact?

Reply:
It still only copies until a value was found since the `Copied` iterator is still lazy.

Reply:
Yes, but it ranges from `1..n` copies, worst case is `n`.

Reply:
Sure, but the point type is only two `f32`s, which is hardly much more expensive than passing a reference.
The scaling behavior wasn't changed either, since neither copying nor taking a reference scales with the length of the vec.

I don't see why the change to copying here would be worse than copying anywhere else.

Reply:
Ok, thumbs up from Schmiddy probably means that we want to keep it as implemented. Approval from my side, knoellle please approve if you are happy.